engine/access/rpc/backend/backend_scripts.go

```diff
@@ -189,7 +187,17 @@ func (b *backendScripts) shouldLogScript(execTime time.Time, scriptHash [16]byte
 	}
 }
 
-func (b *backendScripts) tryExecuteScript(ctx context.Context, executorAddress string, req *execproto.ExecuteScriptAtBlockIDRequest) ([]byte, error) {
+func (b *backendScripts) tryExecuteScriptOnExecutionNode(
+	ctx context.Context,
+	executorAddress string,
+	blockID flow.Identifier,
+	script []byte,
+	arguments [][]byte) ([]byte, error) {
```

Comment:
could you add comma at the end of the arg list (indentation change makes it easier to spot where the signature ends / body begins.  same elsewhere